### ISSUE TITLE
module cannot remove certain limits pam_limits.py

### ISSUE THREAD

**Issue by kerrc** [ansible/ansible-modules-extras#1083]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

pam_limits module
##### ANSIBLE VERSION

N/A
##### SUMMARY

Is it possible to get an ensure: absent component added to this module.

Here I have an example where I want to add limits underneath limits.d as my default but do not want the limit present in limits.conf.

Ideally I want to create a task that will check to see if the value limit_type and limit_item exists in default limits.conf and if so remove it.

Example Run Required:
    pam_limits: domain=user1 limit_type=hard limit_item=nofile backup=yes ensure=abesnt

**Comment by ansibot** [comment]
@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by kerrc** [comment]
Hi - What response are you waiting on from me?
Thanks

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pullrequests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pullrequest to the ansible/ansible repo.

**Comment by kladiv** [comment]
+1
useful to implement "create" too, if destination file different from /etc/security/limits.conf (e.g. /etc/security/limits.d/<file>.conf)

**Comment by felipe1982** [comment]
Hi. Came here from Google looking for how to remove an entry. Tried `state: absent` but complained
```
TASK ********************
fatal: [1.2.2.2]: FAILED! => {"changed": false, "failed": true, "msg": "Unsupported parameters for (pam_limits) module: state. Supported parameters include: backup,comment,dest,domain,limit_item,limit_type,use_max,use_min,value"}
```

**Comment by willthames** [comment]
Please raise a new issue on the main [ansible/ansible](https://github.com/ansible/ansible) repo, this repo is no longer in use.
